testing/kfctl/kfctl_go_test.py

```diff
@@ -128,6 +139,12 @@ def test_build_kfctl_go(app_path, project, use_basic_auth, use_istio):
 
   verify_kubeconfig(project, zone, app_path)
 
+def filterSpartakus(spec):
+  for i, comp in enumerate(spec["components"]):
```

Comment:
Lets use KfDef.Spec.Applications

Reply:
done

Reply:
Did you forget to push changes or does this code need to be updated?

It looks like you updated the specs to use KfDef.Spec.Applications but here you are still looking at spec[components].

It might be good to have filterSpartakus look at both spec["components"] and spec["applications"] this way we filter out spartakus regardless of what format the spec is using.